afs/afs-ws/afs-ws-server/src/main/java/com/powsybl/afs/ws/server/AppStorageServer.java

```diff
@@ -190,6 +190,18 @@ public Response setDescription(@ApiParam(value = "File system name") @PathParam(
         return Response.ok().build();
     }
 
+    @PUT
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("fileSystems/{fileSystemName}/nodes/{nodeId}/setConsistent")
```

Comment:
To rename "fileSystems/{fileSystemName}/nodes/{nodeId}/consistent" to respect resource oriented API ?